src/apis/memberships.js

```diff
@@ -16,32 +17,104 @@ export default (state) => {
   const ROLES = memberships.ROLES = {
     READ: 'read',
     WRITE: 'write',
+    UPLOAD: 'upload',
     ADMIN: 'admin'
   }
 
   /**
    * Add a member to an organization.
    *
    * ```js
-   * rokka.memberships.create('myorg', 'user@example.org', rokka.memberships.ROLES.WRITE)
+   * rokka.memberships.create('myorg', '613547f8-e26d-48f6-8a6a-552c18b1a290', [rokka.memberships.ROLES.WRITE])
    *   .then(function(result) {})
    *   .catch(function(err) {});
    * ```
    *
    * @authenticated
-   * @param  {string}  organization name
-   * @param  {string}  email        email address of a registered rokka user
-   * @param  {string}  role         user role (`rokka.memberships.ROLES`)
+   * @param  {string}        organization name
+   * @param  {string}        userId       email address of a registered rokka user
+   * @param  {string|array}  roles        user roles (`rokka.memberships.ROLES`)
    * @return {Promise}
    */
-  memberships.create = (organization, email, role) => {
-    if (Object.keys(ROLES).map((key) => ROLES[key]).indexOf(role) === -1) {
-      return Promise.reject(new Error(`Invalid role "${role}"`))
+  memberships.create = (organization, userId, roles) => {
+    if (typeof roles === 'string' || roles instanceof String) {
+      roles = [roles]
     }
```

Comment:
is the `roles instanceof String` really necessary?

Reply:
Maybe not, will remove it